packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,38 @@
+import { c, classy, m, ProgressProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value = 0,
+  max = 100,
+  size = 'regular',
+  hideLabel,
+  children,
+  className,
```

Comment:
You also need to pass the rest of props to the parent.

So:

```suggestion
  className,
  ...restProps
```

And then follow the next comment.

Reply:
That's so other props apply, like `data-` and `id` and so on.

Reply:
Done!